src/Board.cpp

```diff
@@ -35,21 +38,35 @@ BLines Board::getBoardLines() const
     return board_;
 }
 
-int Board::getMaxNumbEleInRow() const
+int Board::checkLabelRowsWidth() const
 {
     auto it = std::max_element(rows_.begin(),
                                rows_.end(),
                                [](Line lhs, Line rhs){
```

Comment:
I don't like this formatting, either leave it in one line or put { in the next line like this
```cpp
[](Line lhs, Line rhs)
{
    return lhs.size() < rhs.size();
});
```